cluster/clean.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -ex
+
+echo 'Cleaning up ...'
+
+./cluster/kubectl.sh delete --ignore-not-found -f deploy/
+./cluster/kubectl.sh delete --ignore-not-found -f deploy/crds/nmstate.io_nodenetworkconfigurationenactments_crd.yaml
```

Comment:
one time when Sebastian suggested a way to clean stuff manually
he suggested: (for cnao)
destroy the CR
wait that all needed is deleted
delete the NS 
wait that all needed is deleted
delete the CRD

no need to wait here and such ?

Reply:
It may be better approach to make sure there are no leftovers before re-deploying. However, that would be out of the scope of this PR. We keep the deployment and removal code from the previous version. If we face issues while repeatedly using `make cluster-sync` locally, we can tackle it.